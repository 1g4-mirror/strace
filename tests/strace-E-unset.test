#!/bin/sh
#
# Check the unsetting part of -E option.
#
# Copyright (c) 2016-2018 The strace developers.
# All rights reserved.
#
# SPDX-License-Identifier: GPL-2.0-or-later

. "${srcdir=.}/init.sh"

run_prog ../sleep 0 > /dev/null
export option_E_var=OPTION_E_VAL
export option_E_unset=OPTION_E_UNSET
run_strace -E option_E_unset -v $args
match_grep
