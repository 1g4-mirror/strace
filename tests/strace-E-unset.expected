execve\("\.\./sleep", \["\.\./sleep", "0"\], \[.*"option_E_var=OPTION_E_VAL".*\]\) = 0
!execve\("\.\./sleep", \["\.\./sleep", "0"\], \[.*"option_E_unset=.*\]\) = 0
